x-pack/legacy/plugins/index_management/public/sections/home/index_list/index_table/index_table.js

```diff
@@ -332,6 +347,12 @@ export class IndexTable extends Component {
                 this.toggleItem(name);
               }}
               data-test-subj="indexTableRowCheckbox"
+              aria-label={i18n.translate('xpack.idxMgmt.indexTable.selectIndexAriaLabel', {
+                defaultMessage: 'Select this row'
+              })}
+              title={i18n.translate('xpack.idxMgmt.indexTable.selectIndexTitle', {
```

Comment:
Is title necessary?

Reply:
No, I don't think so. I removed it (can't remember why I originally added it).